### ISSUE TITLE
Allow searching for a profile page

### ISSUE THREAD

**Issue by paulehoffman** [ietf-tools/datatracker#7031]
### Description

I'd like to find the profile page for an individual to see what groups they chair and what their active documents are. That doesn't seem to work under "Document search", nor do I see that under the "Other" menu. This would be particularly useful for people preparing NomCom comments, but also useful in general to get the "context" for someone you have heard of but don't know.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by richsalz** [comment]
We need a general 'search for user' capability, but see #6008  Right now you can find them if you know their email or some such.

**Comment by rjsparks** [comment]
Names work: https://datatracker.ietf.org/person/Robert%20Sparks

**Comment by paulehoffman** [comment]
I think you mean "Names, when correctly encoded in the URL exactly as expected, work". This  proposal is for searching to avoid:
<img width="367" alt="image" src="https://github.com/ietf-tools/datatracker/assets/638939/bcbe844f-9f03-4393-a2db-691750be805b">

**Comment by rjsparks** [comment]
Sure, but in the meantime, for most browsers, you can just type someone's name after /person/ and don't worry about encoding, and the browser will do the right thing for you.

**Comment by richsalz** [comment]
> Sure, but in the meantime, for most browsers, you can just type someone's name after /person/ and don't worry about encoding, and the browser will do the right thing for you.

modulo case, as @paulehoffman was trying to show above.

**Comment by rjsparks** [comment]
No, that's not what Paul was trying to say - he wants search to work from one of the search boxes, not because someone magically knows they can hack a URL.

That said, the URL endpoint is case insensitive:
![image](https://github.com/ietf-tools/datatracker/assets/10996692/a6e58456-decb-47a7-b3fb-a4bd45a21f04)

**Comment by marcblanchet** [comment]
I understand the temporary proposal of typing the name in the url, but this is really not working. Some first and last names are pretty difficult to remember, to type exactly, ...

**Comment by rjsparks** [comment]
@marcblanchet that observation applies to typing names in search boxes too.
What would you be looking for to find people when you can't quite remember their name or its spelling?

**Comment by marcblanchet** [comment]
- well, search is nowadays nice for not needing to type the string exact.
- btw, if we care about bots, we could offer this feature only people are logged in. it seems to me an easy to get around the issue.
